IBPSA/Fluid/CHPs/OrganicRankine/Cycle.mo

```diff
@@ -0,0 +1,494 @@
+        Line(
+          points={{-60,-60},{-28,-20},{16,32},{40,60},{52,60},{54,30},{48,2},{
+              52,-38},{58,-58}},
+          color={255,255,255},
+          smooth=Smooth.Bezier,
+          thickness=0.5),
+        Line(
+          points={{6,20},{52,20},{66,-6},{50,-18},{-26,-18}},
+          color={255,255,255},
+          thickness=0.5,
+          pattern=LinePattern.Dash)}),               Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+Documentation(info="<html>
+<p>
+Model of an organic Rankine cycle (ORC) as a bottoming cycle.
+</p>
+<p>
+The thermodynamic cycle is steady-state while the evaporator and the condenser
+can be configured to have first order dynamics.
+The fluid stream 1 (using <code>Medium1</code>, <code>port_a1</code>, etc.)
+is the evaporator hot fluid carrying waste heat
+and the stream 2 is the condenser cold fluid carrying the cooling fluid.
+The working fluid of the cycle is not based on a typical Modelica medium model.
+See the Thermodynamic Properties section of this document for the rational.
+</p>
+<h4>Cycle Architecture and Governing Equations</h4>
+<p>
+The implemented ORC is modeled based on the simplified cycle shown in the figure below.
+The cycle has two variants depending on the shape of the saturation lines of
+the working fluid and <i>&eta;<sub>exp</sub></i>.
+For any given working fluid, the cycle is fully determined by providing
+the working fluid evaporating temperature <i>T<sub>w,eva</sub></i>,
+the working fluid condensing temperature <i>T<sub>w,con</sub></i>,
+the expander efficiency <i>&eta;<sub>exp</sub></i>, and
+the pump efficiency <i>&eta;<sub>pum</sub></i>.
+The superheating temperature difference <i>&Delta;T<sub>sup</sub></i>
+is minimized, meaning it is zero whenever possible; otherwise it assumes
+the smallest value not to cause the expander outlet state to fall
+under the dome. Subcooling after the condenser is not considered.
+The Thermodynamic Properties section of this document details how these
+state points are found.
+</p>
+<p>
+An important assumption is that all heat is dissipated, i.e.,
+the cycle is not controlled thermal load.
+</p>
+<p align=\"center\">
+<img src=\"modelica://IBPSA/Resources/Images/Fluid/CHPs/OrganicRankine/CycleArchitecture.png\"
+alt=\"CycleArchitecture\" width=\"800\" height=\"300\"/></p>
+<p>
+The cycle processes the heat at a fixed
+<i>T<sub>w,eva</sub></i> provided by the user.
+The evaporator heat exchange is governed by
+</p>
+<p align=\"center\" style=\"font-style:italic;\">
+Q&#775;<sub>eva</sub> = m&#775;<sub>h</sub>&nbsp;c<sub>p,h</sub>&nbsp;(T<sub>h,out</sub> - T<sub>h,in</sub>),<br/>
+Q&#775;<sub>eva</sub> = m&#775;<sub>w</sub>&nbsp;(h<sub>pum,out</sub> - h<sub>exp,in</sub>),
+</p>
+<p>
+where the subscripts are
+<i>eva</i> for evaporator,
+<i>exp</i> for expander,
+<i>h</i> for hot fluid of the evaporator, i.e. the fluid carrying heat,
+<i>pum</i> for pump, and
+<i>w</i> for working fluid.
```

Comment:
I agree. However, as the final conference paper is accepted, I am inclined to keep the figures intact because they are used in the same places. I can revise them if we get a chance to further submit a final version, or when we expand it to a journal paper, in which case the paper reference will be updated.

Reply:
Sounds good. I found them confusing only at first glance